modules/ListView.js

```diff
@@ -432,27 +545,70 @@
           injectionFragment.appendChild(childView.el);
         }
       }, this);
+      this.__updateOrderedModelIdList();
       return $(injectionFragment);
     },
 
     /**
-     * Creates a child view and stores a reference to it
-     * @method __createChildView
-     * @private
-     * @param model {Backbone Model} the model to create the view from
-     * @return {Backbone View} the new child view
+     * Attempts to insert new views and remove stale views individually and correctly reorder all views in an
+     * attempt to be faster then a full view re-render
+     * @method __updateByAddingRemoving
+     * @param oldViews {Array of Views} - correctly ordered list of views before making changes to models to render
+     * @param newViews {Array of Views} - the new views created that will be inserted
+     * @param staleViews {Array of Views} - the stale views that will be removed
      */
-    __createChildView: function(model) {
-      var childView,
-        ChildViewClass = this.childView;
-      if (!_.isFunction(this.childView.extend)) {
-        ChildViewClass = this.childView(model);
+    __updateByAddingRemoving: function(oldViews, newViews, staleViews) {
+      var firstChildViewLeft, injectionSite,
+        view = this,
+        sizeOfOldViews = _.size(oldViews),
+        sizeOfNewViews = _.size(newViews),
+        sizeOfStaleViews = _.size(staleViews);
+      if (view.childrenContainer && sizeOfOldViews && sizeOfOldViews == sizeOfStaleViews) {
+        // we removed all the views!
+        injectionSite = $('<span>');
+        _.first(oldViews).$el.before(injectionSite);
```

Comment:
so this adds the injection site before the old views?

Oh, so you can remove the old views?  You leave them attached so that they take advantage of the jquery move?